libs/service-portal/core/src/lib/navigation/masterNavigation.ts

```diff
@@ -176,6 +176,19 @@ export const servicePortalMasterNavigation: ServicePortalNavigationItem[] = [
         },
       },
 
+      {
+        name: defineMessage({
+          id: 'service.portal:licenses',
+          defaultMessage: 'Skilríki',
+        }),
+        path: ServicePortalPath.LicensesRoot,
+        systemRoute: true,
```

Comment:
Setting this flag to true means to this item will always be included in the navigation whether a route exists for it or not.
This property should probably have been made more descriptive in hindsight.

See https://github.com/island-is/island.is/blob/main/libs/service-portal/core/src/lib/service-portal-core.ts

Reply:
Fixed